content/documentation/pages/4-batch-developer-guides/2-batch/1-simple-task.md

```diff
@@ -44,3 +204,7 @@ As Alana I must ask for an org/space
 ### Kubernetes
 
 Where all the cool kids play.
```

Comment:
Should we remove/comment out the CF and K8s sections?

Reply:
No.  I will be moving the kubernetes and CF sections to this document once this PR is merged.

Reply:
OOOPS.  Yes it will be removed.